scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/msop.yaml

```diff
@@ -0,0 +1,67 @@
+FileHeader:
+  library_Suffix: 'SO'
+  device_type: 'MSOP'
+
+# By: JEFO
+# Email: jacob.overgaard.andersen@gmail.com
+MSOP-16_3x4mm_P0.5mm:
+  size_source: 'http://www.analog.com/media/en/package-pcb-resources/package/pkg_pdf/ltc-legacy-msop/05081669_A_MS16.pdf'
+  body_size_x:
+    nominal: 3
+    tolerance: 0.102
+  body_size_y:
+    nominal: 4
```

Comment:
given as 4.039 in the datasheet

Reply:
Yes, you are in fact correct. This is was "modified" as per suggestion of @Shackmeister, since 4.039 would lead to new 3d-models needed, long naming etc.